src/commands/create_account.js

```diff
@@ -13,7 +13,7 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import cryptoModule from '../utils/cryptoModule';
+import cryptoModule from '../utils/crypto_module';
```

Comment:
How about renaming this file `cryptography` while we're here?

Reply:
I think it would be great, but should we do it in this PR?

Reply:
This issue is about standardizing file names, I think it's covered.

Reply:
ok ill change the PR title too